nautobot/ipam/models.py

```diff
@@ -948,9 +983,18 @@ class Meta:
         ordering = ("host", "prefix_length")  # address may be non-unique
```

Comment:
Perhaps?

```suggestion
        ordering = ("parent__namespace__name", "ip_version", "host", "prefix_length")
```

Actually, do we even need to order by prefix_length now that host is unique within a namespace?

Reply:
Eek I am hesitant to do default, ordering based on namespace merely since that's a table join that won't scale well.

Also I'm thinking that since the addresses are all hierarchically numeric that , v4 and v6 should naturally sort without the need for `ip_version` to be part of ordering.

Lastly, yeah `prefix_length` needs to be reconsidered overall.

Reply:
> So if we were to use `namespace` then its ordering would also be asserted, which is by `name`.

I need to dig a bit more into this then, because `Prefix` orders by `namespace` but in my local testing at least, prefixes in namespace `Gamma` are displayed after those in namespace `Global` when using default sorting, which seems wrong to me.

Reply:
Ah. Actually `Namespace` doesn't have any defined `ordering` at present, or even a `Meta` at all. I'll add that as part of #3496 if that's OK with you.

Reply:
In my mind I had added ordering by name. 😑